pytest_vcr.py

```diff
@@ -84,7 +84,20 @@ def vcr_cassette(request, vcr, vcr_cassette_name):
     """Wrap a test in a VCR.py cassette"""
     kwargs = {}
     _update_kwargs(request, kwargs)
+
+    refresh = False
+    if kwargs.get("record_mode") == "refresh":
+        refresh = True
+        kwargs["record_mode"] = 'all'
+
     with vcr.use_cassette(vcr_cassette_name, **kwargs) as cassette:
+        if refresh:
+            # force a cassette rewrite
+            cassette.data = []
+            try:
+                os.remove(cassette._path)
```

Comment:
What happens if the file is removed and then the regeneration of the file fails ?